src/behaviors/ExpandToggle/FixAuto.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AnimatedValue } from 'react-spring';
+
+const getValues = object => Object.keys(object).map(k => object[k]);
```

Comment:
`Object.values()`?

Reply:
Ah, yeah, I just copied their code and patched it, so it's not all idiomatic.